### DISCUSSION TITLE
How to plotting discrete inverse pole figures using iq value as colorkey

### DISCUSSION THREAD

**Discussion by Diego8901** [mtex-toolbox/mtex#834] [Ask Anything] [answered]
Dear all,

I am trying to plot the individual orientations of an EBSD data into Pole figures and inverse pole figures. I would like to set the color of the individual point according to their iq value. 
For the Pole figures, it works by using the following code:

h = [Miller(1,0,0,ebsd.CS),Miller(1,1,0,ebsd.CS)];
plotPDF(ebsd.orientations,ebsd.iq,h,'antipodal','MarkerSize',4)

but for IPF, I tried the following code, it does work. All the points always present the same color, and the iq values of the points were not reflected by the color of the corresponding point.  

plotIPDF(ebsd.orientations,xvector,ebsd.iq,'antipodal','MarkerSize',4) 

Can anybody help me with this?

Many thanks!

Diego

**Comment by ralfHielscher** [comment, marked as answer]
The syntax is the same as for pole figures. You just swapped the order of the argumets:
```matlab
plotIPDF(ebsd.orientations,ebsd.iq,xvector,'antipodal','MarkerSize',4)
```

**Reply by Diego8901** [reply]
Thanks, Ralf. 

It works.
